Update README.md (#57)
adulbrich/ephira: README.md

- [iOS simulator](https://docs.expo.dev/workflow/ios-simulator/)
- [Expo Go](https://expo.dev/go), a limited sandbox for trying out app development with Expo

> You can start developing by editing the files inside the **app** directory. This project uses [file-based routing](https://docs.expo.dev/router/introduction). 

Reviewer: ```suggestion
This project uses [file-based routing](https://docs.expo.dev/router/introduction). 
```

### Learn more

> To learn more about developing your project with Expo, look at the following resources:
> 
> - [Expo documentation](https://docs.expo.dev/): Learn fundamentals, or go into advanced topics with our [guides](https://docs.expo.dev/guides).
> - [Learn Expo tutorial](https://docs.expo.dev/tutorial/introduction/): Follow a step-by-step tutorial where you'll create a project that runs on Android, iOS, and the web.

Reviewer: ```suggestion
To learn more about the libraries we use, look at the following resources:

- [Expo](https://docs.expo.dev/)
- [React Native](https://reactnative.dev/)
- [React Native Paper](https://reactnativepaper.com/)
- [Drizzle (Expo SQLite)](https://orm.drizzle.team/docs/connect-expo-sqlite)
```

> ### Join the community
> 
> Join our community of developers creating universal apps.
> 
> - [Expo on GitHub](https://github.com/expo/expo): View our open source platform and contribute.
> - [Discord community](https://chat.expo.dev): Chat with Expo users and ask questions.

Reviewer: ```suggestion
```
